integration-web/pom.xml

```diff
@@ -51,6 +51,12 @@
             <artifactId>integration-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
```

Comment:
I don't think this import is required, once you have already imported this module into 'integration-core'